src/config/configJsonSchema.js

```diff
@@ -36,13 +39,50 @@ module.exports = {
           type: 'object',
           properties: {
             port: {
-              type: 'integer',
-              minimum: 0,
+              $ref: '#/definitions/port',
+            },
+            seeds: {
+              type: 'array',
+              items: {
+                type: 'object',
+                properties: {
+                  host: {
+                    type: 'string',
+                    minLength: 1,
+                  },
+                  port: {
+                    $ref: '#/definitions/port',
+                  },
+                },
+                required: ['host', 'port'],
+                additionalProperties: false,
+              },
+            },
+          },
+          required: ['port', 'seeds'],
+          additionalProperties: false,
+        },
+        rpc: {
+          type: 'object',
+          properties: {
+            port: {
+              $ref: '#/definitions/port',
+            },
+            user: {
+              type: 'string',
+              minLength: 1,
+            },
+            password: {
+              type: 'string',
+              minLength: 1,
             },
           },
-          required: ['port'],
+          required: ['port', 'user', 'password'],
           additionalProperties: false,
         },
+        sporkAddr: {
```

Comment:
Hate shortcuts. It seems we need `spork` object with `address` and `privateKey` properties.

Reply:
Added